man/rosenpass.1

```diff
@@ -0,0 +1,115 @@
+.Nm
+performs cryptographic key exchanges that are secure against quantum-computers
+and outputs the keys.
+These keys can then be passed to various services such as WireGuard or other
+VPN services as pre-shared-keys to achieve security against attackers with
+quantum computers.
+.Pp
+This is a research project and quantum computers are not thought to become
+practical in less than ten years.
+If you are not specifically tasked with developing post-quantum secure systems,
+you probably do not need this tool.
+.Pp
+Each
+.Ar COMMAND
+may expect further
+.Ar OPTIONS
+and
+.Ar ARGS .
+The commands are as follows:
+.Bl -tag -width Ds
+.It Ar keygen private-key <file-path> public-key <file-path>
+Generates a keypair to use in the exchange command later.
+Send the public-key file to your communication partner and keep the private-key
+file a secret!
+.It Ar exchange private-key <file-path> public-key <file-path> [OPTIONS] PEERS
+Start a process to exchange keys with the specified peers.
+You should specify at least one peer.
+.Pp
+The
+.Ar OPTIONS
+are as follows:
+.Bl -tag -width Ds
+.It Ar listen <ip>[:<port>]
+Instructs
+.Nm
+to listen on the specified interface and port.
+By default
+.Nm
+will listen on all interfaces and select a random port.
+.It Ar verbose
+Enable extra logging.
+.El
+.Pp
+Each
+.Ar PEER
+is defined as follows:
+peer public-key <file-path> [endpoint <ip>[:<port>]]
+[preshared-key <file-path>] [outfile <file-path>] [wireguard <dev> <peer>
+<extra_params>]
+.Pp
+It instructs
+.Nm
+to exchange keys with the given peer and write the resulting PSK into the given
+output file.
+You must either specify the
+.Ar outfile
+or WireGuard output options.
+.Pp
+The parameters of
+.Ar PEER
+are as follows:
+.Bl -tag -width Ds
+.It Ar endpoint <ip>[:<port>]
+Specifies the address where the peer can be reached.
+This will be automatically updated after the first successful key exchange with
+the peer.
+If this is unspecified, the peer must inititate the connection.
+.It Ar preshared-key <file-path>
+You may specify a pre-shared key which will be mixed into the final secret.
+.It Ar outfile <file-path>
+You may specify a file to write the exchanged keys to.
+If this option is specified,
+.Nm
+will write a notification to standard out, every time the key is updated.
+.It Ar wireguard <dev> <peer> <extra_params>
+This allows you to directly specify a WireGuard peer to deploy the
+pre-shared-key to.
+You may specify extra parameters you would pass to
+.Xr wg 1
+besides the preshared-key parameter which is used by
+.Nm .
+This makes it possible to add peers entirely from
+.Nm .
+.El
+.El
+.Sh EXIT STATUS
+.Ex -std
+.Sh SEE ALSO
+.Xr rp 1 ,
+.Xr wg 1
+.Sh STANDARDS
+This tool is the reference implementation of the Rosenpass whitepaper, written
+by Karolin Varner, Benjamin Lipp, Wanja Zaeske, and Lisa Schmidt.
+.Sh AUTHORS
+Rosenpass was created by Karolin Varner, Benjamin Lipp, Wanja Zaeske,
+Marei Peischl,Stephan Ajuvo, and Lisa Schmidt.
```

Comment:
maybe put a space after the comma? And I believe Ajuvo is not the surname, @ajuvo how do you want to be mentioned here?